### PR TITLE
Add toggle for pose markers to submaps display RViz plugin.

### PR DESCRIPTION
**Pull request by MichaelGrupp**
* toggle all submap pose markers
* or toggle per trajectory

### CODE REVIEW

**Comment by MichaelGrupp** [comment]
A GIF says more than 1000 lines of code:

File: cartographer_rviz/cartographer_rviz/submaps_display.cc [hunk 1]
```diff
@@ -256,10 +272,24 @@ void Trajectory::AllEnabledToggled() {
...
+
+Trajectory::Trajectory(std::unique_ptr<::rviz::BoolProperty> property,
+                       const bool pose_markers_enabled)
     : visibility(std::move(property)) {
   ::QObject::connect(visibility.get(), SIGNAL(changed()), this,
                      SLOT(AllEnabledToggled()));
+  pose_markers_visibility = absl::make_unique<::rviz::BoolProperty>(
+      QString("Submap Pose Markers"), pose_markers_enabled,
+      QString("Toggles the submap pose markers of this trajectory."),
+      visibility.get());
```
Comment (gaschler) [inline cartographer_rviz/cartographer_rviz/submaps_display.cc]: why visibility.get()? shouldn't this be pose_markers_enabled?

Comment (MichaelGrupp) [reply cartographer_rviz/cartographer_rviz/submaps_display.cc]: oh yes.

Comment (MichaelGrupp) [reply cartographer_rviz/cartographer_rviz/submaps_display.cc]: edit: no ;) it's a bit confusing, but this means that the toggle for this trajectory is in the submaps visibility list (the drop down thing I also open in the GIF)

Comment (MichaelGrupp) [reply cartographer_rviz/cartographer_rviz/submaps_display.cc]: Added a comment to make it clear.

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user gaschler. Merge queue now has a length of 1.

**Comment by wally-the-cartographer** [comment]
Refusing to merge, since the combined checks state is failure

**Comment by MichaelGrupp** [comment]
@gaschler Can you manually re-run that one build that timed out on Travis?

**Comment by MichaelGrupp** [comment]
@gaschler Build has finally passed.

**Pull request merged**